pkgs/top-level/all-packages.nix

```diff
@@ -13074,8 +13074,6 @@ in
 
   liblastfmSF = callPackage ../development/libraries/liblastfmSF { };
 
-  liblastfm = libsForQt5.callPackage ../development/libraries/liblastfm { };
```

Comment:
Could use an alias in top-level/aliases.nix

Reply:
Done.